pulp_2to3_migration/app/migrations/0025_remove_p2content_dups.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.19 on 2021-03-09 01:47
+
+from django.db import migrations, models
+
+
+def fix_dups(apps, schema_editor):
+    pulp2content = apps.get_model("pulp_2to3_migration", "Pulp2Content")
+    # find sets of p2-id/p2-type/-2-repo/p2-sub dups
+    p2content = pulp2content.objects.values('pulp2_id', 'pulp2_content_type_id', 'pulp2_repo_id', 'pulp2_subid').order_by('-pulp2_id')
+    p2content = p2content.annotate(count_id=models.Count("pulp2_id")).filter(count_id__gt=1)
+
+    # for each set of dups...
+    for dup in p2content:
```

Comment:
```.iterator()```

Reply:
I guess it's not a huge deal unless there's a TON of duplicates

Reply:
it should be very few duplicates if any. It happens when pulp2 content (not errata) changes and it's only for some fields like `downloaded`.

Reply:
Also, the only people who can be in this situation right now is "us". I'm OK with leaving it like this.